### PR TITLE
Task nxp 32775 lts 2025

### PR DESCRIPTION
**Pull request by guirenard**

### CODE REVIEW

**Comment by guirenard** [comment]
Just for review, we won't merge this PR, we'll just rename the branch as `lts-2025`

File: pom.xml [hunk 1]
```diff
@@ -4,13 +4,13 @@
   <parent>
     <groupId>org.nuxeo</groupId>
     <artifactId>nuxeo-parent</artifactId>
-    <version>2023.1</version>
+    <version>2025.0.31</version>
```
Comment (kevinleturc) [inline pom.xml]: Maybe better to leverage `2025.0-SNAPSHOT` until we release 2025.

**Review by kevinleturc — DISMISSED** [review]

File: pom.xml (hunk 1 above)
Comment (guirenard) [reply pom.xml]: yes indeed

**Comment by guirenard** [comment]
So we agree the plan is to remove https://github.com/nuxeo/nuxeo-retention/pull/180/commits/746cd46378d76f0cb1949cd92d301210288e9594, rename the branch `lts-2025` and that's all?

**Comment by kevinleturc** [comment]
> So we agree the plan is to remove [746cd46](https://github.com/nuxeo/nuxeo-retention/commit/746cd46378d76f0cb1949cd92d301210288e9594), rename the branch `lts-2025` and that's all?

Yes

**Pull request closed without merging**